src/com/comeet/UserService.java

```diff
@@ -120,6 +120,47 @@
 
 
     /**
+     * Implementing method for GET /{organization}/meetings/attendees?id=
+     * <p>
+     * Example URL:
+     * </p>
+     * 
+     * @param orgDomain Start of query range.
+     * @param id - meeting ID
+     * @return Meeting info within the query range.
+     * @throws ServiceResponseException If the result is not 200 OK
+     * @throws Exception On an unexpected error.
+     */
+    @POST
+    @Path("/{orgDomain}/meetings/attendees")
+    @Produces("application/json")
+    @Consumes(MediaType.APPLICATION_FORM_URLENCODED)
+    public Attendees getAttendees(@Context HttpHeaders headers,
+                    @PathParam("orgDomain") String orgDomain, 
+                    @DefaultValue("") @FormParam("id") String id)
+                    throws ServiceResponseException, Exception {
+
+        try {
+            serviceFactory.setAuthContext(authFactory.buildContext(headers));
+
+            // Each call should define new instance of Service and DAO object
+            try (ExchangeService service = serviceFactory.create()) {
+                UsersDao ud = new UsersDao(service);
+                return ud.getAttendees(id);
+            }
+        } catch (AuthContextException e) {
+            // TODO Use OAuth2 for real.
+            e.printStackTrace();
```

Comment:
I meant instead of printStackTrace not in addition to. you can change it in another pull request